e2e/replay/msg_persister.go

```diff
@@ -86,23 +86,23 @@ func (r *replayMessagePersister) createFile() error {
 	return nil
 }
 
-// closeFile closes file created by the ReplayMessagesHandler if it is open
-func (r *replayMessagePersister) closeFile() error {
+// CloseFile closes file created by the ReplayMessagesHandler if it is open
+func (r *MessagePersister) CloseFile() error {
 	if r.file != nil {
 		return r.file.Close()
 	}
 	return nil
 }
 
-// addMessage adds a message from sequence to messages cache that will be written to .flow file
-func (r *replayMessagePersister) addMessage(message *ReplayMessage) {
+// AddMessage adds a message from sequence to message cache that will be written to .flow file
+func (r *MessagePersister) AddMessage(message *Message) {
```

Comment:
Seems like this doesn't need to be public eventually (it is called by `MessageNotifier`, which is in the `replay` package as well)